target_decisioning_engine/types/target_delivery_request.py

```diff
@@ -13,12 +13,11 @@
 class TargetDeliveryRequest:
     """TargetDeliveryRequest"""
 
-    def __init__(self, request, visitor_cookie=None, target_cookie=None, target_location_hint=None,
```

Comment:
I see the visitor_cookie has been removed. Are we going to add support for server-side Visitor library and add AMCV cookie parsing to the sample app in a future PR?

Reply:
I'm not sure about that.  Product decision is to do initial release without Visitor library, and I'm not sure the timeline for adding that support.